packages/backend/src/services/SchedulerService/SchedulerService.ts

```diff
@@ -425,4 +443,59 @@ export class SchedulerService extends BaseService {
             undefined,
         );
     }
+
+    async updateDefaultTimezoneSchedulers(
```

Comment:
Small thing, but the name similarity between `updateDefaultTimezoneSchedulers` and `updateDefaultSchedulerTimezone` is a little confusing. Could this one be `updateSchedulersWithDefaultTimezone`? Ignore if you disagree :D

Reply:
Updated 👌

Reply:
Also, bug found was fixed in [fix: timezone being passed as null](https://github.com/lightdash/lightdash/pull/12015/commits/452fd87b2a2535b6afb7dd88bd5747d55badf9d3)

Reply:
Yup, fixed